tests/performance-test/README.md

```diff
@@ -3,18 +3,139 @@
 ## Introduction
 
 The performance test provides an automated environment in which to to run stress
-tests on the SAF locally using Minishift. Collectd-tg is used to simulate
-extensive netwrok traffic to pump through SAF. Because Minishift only supports a
-single node at a time, this test demonstrates the limits of SAF in a constrained
-environment. Test scenarios are manually configured in a yaml file and results
-can be analyzed in a series of grafana dashboards.
+tests on SAF. Collectd-tg or telemetry-bench are used to simulate extensive
+metrics data to pump through SAF. Results of testing can be analyzed in a
+grafana dashboard.
 
 Two additional pods are deployed by the performance test: one that hosts a
 grafana instance and one that executes the testing logic.
 
 ![A Performance Test Dashboard](images/dashboard.png)
 
-## Configuring Tests
+## Environment
+
+* openshift v3.11.135
+* docker v17.05+
+
+## Setup
+
+SAF must already be deployed, a quick way to do this is using the
+`quickstart.sh` script in `telemetry-framework/deploy/` directory to run SAF.
+Here is an example of how to do that in minishift:
+
+>NOTE: if you intend to rebuild any test containers, your openshift must have an
+>external route for the registry. The command for that is included here.
+
+```shell
+minishift addons enable registry-route   # Run BEFORE starting minishift
+minishift start
+eval $(minishift oc-env)
+cd $WORKDIR/telemetry-framework/deploy/; ./quickstart.sh
+```
+
+More details about deploying SAF can be found in the
+[SAF deployment docs](../../../docs/README.adoc#quickstart-script-for-development-usage)
```

Comment:
I was gonna say you can link to the upstream document but that part isn't imported into upstream `saf-documentation` yet... I'll have to create an issue in `saf-documentation` to migrate this path once it's available.